.github/workflows/lychee_url_checker.yml

```
@@ -0,0 +1,27 @@
+name: Check URLs with Lychee
+
+on:
+  push:
+  pull_request:
+  schedule:
+    # Run everyday at 3 am UTC
+    - cron: 0 3 * * *
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Lychee URL checker
+      # use stable version for now to avoid breaking changes
+      # sha-256 hash of latest stable version for security reasons
+        uses: lycheeverse/lychee-action@d322e38893906cef8788eefe9c63694c562c7dfa
```

Also, I think it should be okay to leave it as `@v1`. These actions are usually safe. (Given that `@v1` works, I think it should.)
```suggestion
        uses: lycheeverse/lychee-action@v1
```

---

Maybe I can pin it to v1.6.1 instead because it has been updated to use the latest `lycheeVersion` by default, i.e., v0.11.1